Tools/machines/hpc3-uci/hpc3_gpu.sbatch

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -l
+
+# Copyright 2023 The WarpX Community
+#
+# This file is part of WarpX.
+#
+# Authors: Axel Huebl, Victor Flores
+# License: BSD-3-Clause-LBNL
+
+#SBATCH --time=00:30:00
+#SBATCH --nodes=1
+#SBATCH -J WarpX
+#S BATCH -A <proj>
+# V100 GPU options: gpu, free-gpu, debug-gpu
+#SBATCH -p free-gpu
+#SBATCH --cpus-per-task=10
+#SBATCH --gres=gpu:V100:4
```

Comment:
@floresv299 we need to test if this properly uses all four GPUs. I have not yet found so-called Slurm "pinning" options, where we assign the MPI rank to the CPUs that are closest to each of the four GPUs. Something we need to:
- test by running 1, 2, 4 GPUs (1 node) and 8 GPUs (2 nodes) and
- need to write an email to your HPC3 support about :)

Reply:
I sent an email to UCI HPC support to square out the last question on system-specific control of GPU affinity.

@floresv299 a small wish to easier identify you on GitHub: Do you mind setting your name and affiliation in your profile https://github.com/settings/profile ? :)

Reply:
Sure thing Axel, just did. Thank you once again for all your help, you have been amazing !

Reply:
Thanks, you are most welcome!